Fix parameter name in tl_page::getTitleTag()

SeverinGloeckle opened the pull request:
This fixes a wrong parameter name in `tl_page::getTitleTag()`, which probably got introduced when merging 4.12 into 4.x in https://github.com/contao/contao/commit/646b4b792c8e3ba474d04768056bf1af4c98cd9e on top of the changes from https://github.com/contao/contao/commit/e3b473b10a63254a2f9f2d20dbc18315b5b0db74.

I decided to keep the parameter name `$model` instead of `$page` (from https://github.com/contao/contao/commit/e3b473b10a63254a2f9f2d20dbc18315b5b0db74) for consitency with similar methods like `tl_page::getSerpUrl)`, `tl_news::getTitleTag()`.

bytehead reviewed (dismissed):
Nice catch! 👀  Static analysis would cover that 🙈

leofeyer committed:
Rename $model to $page

SeverinGloeckle committed:
use $page instead of $model consistently

SeverinGloeckle commented:
As I understand, @leofeyer prefers to name the PageModel `$page`. I updated the method's PHPDocBlock accordingly and also changed the parameter name in `tl_page::getSerpUrl()` from `$model` to `$page`  for consistency.

leofeyer commented:
It is not just that I want it and therefore it has to be done. 😄 The parameter is called `$page` in Contao 4.9 
and to ensure that there are no conflicts when merging upstream, it should be called `$page` in Contao 4.13 as well.

https://github.com/contao/contao/blob/bfe5629e8e88147e9adf6ad0922712ccf24896db/core-bundle/src/Resources/contao/dca/tl_page.php#L1006-L1014

I did not want to bother you with this change, though.

leofeyer commented:
BTW, we should also fix this in Contao 4.12. @SeverinGloeckle Can you rebase the PR on the `4.12` branch please?

leofeyer commented:
Thank you @SeverinGloeckle.
